src/flexmeasures_client/s2/control_types/FRBC/frbc_simple.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from datetime import timedelta
+
+from python_s2_protocol.FRBC.messages import FRBCSystemDescription
+
+from flexmeasures_client.s2.control_types.FRBC import FRBC
+from flexmeasures_client.s2.control_types.FRBC.utils import fm_schedule_to_instructions
+
+
+class FRBCSimple(FRBC):
+    _power_sensor_id: int
+    _price_sensor_id: int
+    _schedule_duration: timedelta
+
+    def __init__(
+        self,
+        power_sensor_id: int,
+        price_sensor_id: int,
+        schedule_duration: timedelta = timedelta(hours=12),
+        max_size: int = 100,
+    ) -> None:
+        super().__init__(max_size)
+        self._power_sensor_id = power_sensor_id
+        self._price_sensor_id = price_sensor_id
+        self._schedule_duration = schedule_duration
+
+    async def trigger_schedule(self, system_description_id: str):
+        """Translates S2 System Description into FM API calls"""
+
+        system_description: FRBCSystemDescription = self._system_description_historic[
+            system_description_id
+        ]
+
+        # fill_level_range = system_description.storage.fill_level_range
+        soc_at_start = 0.221  # TODO: take the most recent
+
+        # call schedule
+        schedule_id = await self._fm_client.trigger_storage_schedule(
+            start=system_description.valid_from,  # TODO: localize datetime
+            sensor_id=self._power_sensor_id,
+            production_price_sensor=self._price_sensor_id,
+            consumption_price_sensor=self._price_sensor_id,
+            soc_unit="MWh",
```

Comment:
Let's collect this information centrally, too. It seems S2 only diagnostically describes the fill level unit in `storage_description.fill_level_label`, but I think we may need to use it to define a mapping between the `soc_unit` (in FM) and `fill_level_label` (in S2).

Reply:
Yes, using `fill_level_label` is a possibility although, I'm not sure this field is intended for this purpose:

> Human readable description of the (physical) units associated with the fill_level (e.g. degrees Celsius or percentage state of charge). **This element is only intended for diagnostic purposes and not for HMI applications.**